apps/web/app/s/[surveyId]/LinkSurvey.tsx

```diff
@@ -50,7 +50,10 @@ export default function LinkSurvey({ survey }: LinkSurveyProps) {
           loadingElement && "animate-pulse opacity-60",
           "flex h-full flex-1 items-center overflow-y-auto bg-white"
         )}>
-        <ContentWrapper className="w-full md:max-w-lg">
+        <ContentWrapper className={cn(
+          isPreview && "mt-[44px]",
```

Comment:
Please explain what you're solving here and why you chose this approach.

Reply:
@jobenjada In Preview mode, there is a preview strip at the top and due to this some portion of the content get hidden below this preview strip
![image](https://github.com/formbricks/formbricks/assets/51370246/e12297d3-5ad2-4577-a88d-f1a191af4c1c)